python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -291,9 +314,18 @@ async def _send_chat_stream_request(
     # endregion
     # region content creation
 
+    def _settings_messages_to_prompt(self, messages: list[dict[str, Any]]) -> str:
```

Comment:
Why do we need this method, it is already a dict with role and content?

Reply:
This only keeps two fields from the entries in the input dict.  Is there a better way to do this?